src/reducers/tags.js

```diff
@@ -77,6 +78,16 @@ export const tagsByCommunity = (state = {}, action) => {
     case FETCH_CURRENT_USER:
     case CREATE_COMMUNITY:
       return mergeLeftNavTags(state, get('left_nav_tags', payload))
+    case UPDATE_POST_EDITOR:
+      const { tagDescriptions } = payload
+      let { slug } = meta
```

Comment:
just put `let slug` up before the switch statement rather than having the `let` here affect the assignments further on in the file